resources/startup.sh

```diff
@@ -190,6 +249,55 @@ function subsequentSonarStart() {
   sed -i "/sonar.cas.casServerLogoutUrl=.*/c\sonar.cas.casServerLogoutUrl=https://${FQDN}/cas/logout" ${SONAR_PROPERTIESFILE}
 
   setProxyConfiguration
+
+
+  # start sonar in background to have the possibility to import quality profiles
+  su - sonar -c "java -jar /opt/sonar/lib/sonar-application-$SONAR_VERSION.jar" &
+
+  END=$((SECONDS+120))
```

Comment:
Use [doguctl wait-for-http](https://github.com/cloudogu/doguctl#wait-for-http) functionality for this